modules/ocf/manifests/packages/firefox.pp

```diff
@@ -3,14 +3,20 @@
     stage => first,
   }
 
-  $firefox_pkg = $::lsbdistcodename ? {
-    'jessie' => 'firefox',
-    default  => 'firefox-esr',
+  if $::lsbdistcodename == 'jessie' {
```

Comment:
Alright, I didn't notice that before, so that is good to know. I still think it would make things a lot easier though, even if it has a cronjob and apt hook. The issue with using `apt-cache policy` is that it shows all the sources that it could be installed from, not just the one it actually is installed from, which sounds a lot harder to work with.